### PR TITLE
Contacts with protocol = AP are now delivered only via AP

### PR DESCRIPTION
**Pull request by annando**
When two newer Friendica contacts do connect, then this is done via AP (see my recent PR). With this PR we now ensure that the message delivery is only done via AP to these contacts. (Except some currently unsupported or barely tested things)

### CODE REVIEW

File: src/Worker/Notifier.php [hunk 1]
```diff
@@ -388,10 +388,11 @@ function_exists('imap_open')
 
 				$relay_list_stmt = DBA::p(
 					"SELECT
-       					`batch`,
-       					ANY_VALUE(`id`) AS `id`,
-       					ANY_VALUE(`name`) AS `name`,
-       					ANY_VALUE(`network`) AS `network`
+						`batch`,
```
Comment (tobiasd) [inline src/Worker/Notifier.php]: Why the change of indentation? Readability as the indented part is the "select" part of the query?

Comment (annando) [reply src/Worker/Notifier.php]: I just had replaced the spaces with tabs.

Comment (tobiasd) [reply src/Worker/Notifier.php]: Ah, there at the beginning of the line :) I had only checked the last indentation steps before the code.
